packages/opc-footer/src/opc-footer.ts

```diff
@@ -1,12 +1,11 @@
 import { LitElement, html, property, customElement } from 'lit-element';
 import style  from './opc-footer.scss';
-import { OpcFooterLinksGroup } from './opc-footer-interface';
 
 @customElement('opc-footer')
 export class OpcFooter extends LitElement {
   @property({ attribute: 'opc-theme' }) theme: string = 'light';
 
-  @property({ type: Array, attribute: 'opc-links-group' })
+  @property({ type: Array, attribute: 'opc-links' })
   linksGroups: OpcFooterLinksGroup[] | undefined = [];
```

Comment:
if you're changing the name in the property then make the change everywhere @hybridx @gisumit

Reply:
@diwanshi should we make it to just `links` ?